apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts

```diff
@@ -29,6 +29,25 @@ export class BookingsRepository_2024_08_13 {
     });
   }
 
+  async getByIdsWithAttendeesWithBookingSeatAndUserAndEvent(ids: number[]) {
+    return this.dbRead.prisma.booking.findMany({
+      where: {
+        id: {
+          in: ids,
+        },
+      },
+      include: {
+        attendees: {
+          include: {
+            bookingSeat: true,
+          },
+        },
+        user: true,
```

Comment:
are we formatting the password out of the response data ?

Reply:
seems to be done in output service 👍

Reply:
Yes - we have `@Expose()` and in output service we use `plainToClass` with corresponding class.